app/database/group_attempt_store_compute_all_group_attempts.go

```diff
@@ -101,7 +101,8 @@ func (s *GroupAttemptStore) ComputeAllGroupAttempts() (err error) {
 							MAX(aggregated_children_attempts.validated_at) AS max_validated_at,
 							MAX(IF(items_items.category = 'Validation', aggregated_children_attempts.validated_at, NULL))
 								AS max_validated_at_categories,
-							IFNULL(AVG(IFNULL(aggregated_children_attempts.score, 0) * items_items.score_weight), 0) AS average_score
+							IFNULL(SUM(IFNULL(aggregated_children_attempts.score, 0) * items_items.score_weight), 0) /
+								COALESCE(NULLIF(IFNULL(SUM(items_items.score_weight), 1), 0), 1) AS average_score
```

Comment:
Same here, how can `SUM(items_items.score_weight)` be null with a `FROM items_items` and a score_weight which cannot be null?
Then, why does the `COALESCE` set "1/100" as score if the sum of weight is 0? Where does "1" come from?

Reply:
When the sum of weights is 0, the nominator is 0 too (because this means all the weights are zeros since weights cannot be negative). In this situation we use as the denominator to prevent division by 0.

Reply:
Ok so you compute "0/1" which equals "0". Why not... ok.

There were 2 questions. The first one is unanswered:
"how can SUM(items_items.score_weight) be null with a FROM items_items and a score_weight which cannot be null?"

Reply:
SUM(items_items.score_weight) is null when there are no items_items matching the criteria.

Reply:
How can it be when the query is on items_items?

Reply:
```

Reply:
```

Reply:
```
mysql> select sum(id) from items_items group by parent_item_id;
Empty set (0.00 sec)
```

Reply:
ok. fixed